### DISCUSSION TITLE
[x][closed][duplicate] EGAXs not referenced by EGA services? EGAX Curie not supported in identifiers.org.

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#108] [Team Posts]
Response from EGA team: Interesting, will look into it later. Updates?

**Comment by dzerbino** [comment]
No, sorry.

**Comment by jmfernandez** [comment]
Only to keep it tracked here, as I told you in a TC I wrote Jordi Rambla a few weeks ago about this matter. He answered he was going to write proper people to reactivate the topic, but no news yet.

**Comment by sveinugu** [comment]
Saw now that this is a duplicate of https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/22. Closing this.
